### PR TITLE
Loc Update

### PR DESCRIPTION
**Pull request by cristianosuzuki77**
Latest translations (including fixes to previous translations).

### CODE REVIEW

File: src/Microsoft.CodeAnalysis.Analyzers/Core/xlf/CodeAnalysisDiagnosticsResources.es.xlf [hunk 1]
```diff
@@ -24,12 +24,12 @@
       </trans-unit>
       <trans-unit id="CompareSymbolsCorrectlyDescription">
         <source>Symbols should be compared for equality, not identity. Use an overload accepting an 'IEqualityComparer' and pass 'SymbolEqualityComparer'.</source>
-        <target state="needs-review-translation">Solo se debe realizar una comparación de igualdad de los símbolos, no de identidad.</target>
+        <target state="translated">Debe compararse la igualdad de los símbolos, no la identidad. Use una sobrecarga que acepte un elemento "IEqualityComparer" y pase "SymbolEqualityComparer".</target>
```
Comment (Youssef1313) [inline src/Microsoft.CodeAnalysis.Analyzers/Core/xlf/CodeAnalysisDiagnosticsResources.es.xlf:27]: Relates to #4533

**Comment by mavasani** [comment]
@buyaa-n The Spanish unit test failures here are for PlatformCompat analyzer. Can you please take a look and fix them? Let us know if you prefer skipping these tests in this PR + filing a follow-up issue OR you'd like to fix them up in this PR itself.

**Comment by buyaa-n** [comment]
> @buyaa-n The Spanish unit test failures here are for PlatformCompat analyzer. Can you please take a look and fix them? Let us know if you prefer skipping these tests in this PR + filing a follow-up issue OR you'd like to fix them up in this PR itself.

Sure looking into it

**Comment by buyaa-n** [comment]
@mavasani @cristianosuzuki77 sorry i was not able to push to this PR (cannot push `cristianosuzuki77/master` branch directly), created a PR https://github.com/cristianosuzuki77/roslyn-analyzers/pull/1/files. Maybe raise a PR into `dotnet/roslyn-analyzers `instead

**Comment by buyaa-n** [comment]
@cristianosuzuki77 @mavasani test failures fixed. I think you can merge it now

**Review by mavasani — APPROVED** [review]

**Pull request merged**